namerd/iface/control-http/src/test/scala/io/buoyant/namerd/iface/HttpControlServiceConfigTest.scala

```diff
@@ -51,13 +51,15 @@ class HttpControlServiceConfigTest extends FunSuite {
   }
 
   test("socket options"){
-    val expectedOpts = SocketOptionsConfig(reusePort = true)
+    val expectedOpts = SocketOptionsConfig(reusePort = true, readTimeoutSeconds = Some(60), keepAlive = Some(true))
     val yaml = """
       |kind: io.l5d.httpController
       |socketOptions:
       |  noDelay: true
       |  reuseAddr: true
       |  reusePort: true
+      |  readTimeoutSeconds: 60
+      |  keepAlive: true
```

Comment:
Thanks for modifying the test with the new change! For completeness, I think we should also include `writeTimeoutSeconds` as well.